ExpressBindingGenerator/src/Generator/GeneratorOIP.cpp

```diff
@@ -4102,13 +4124,11 @@ void GeneratorOIP::generateEntitySourceFileREFACTORED(Schema & schema, const Ent
 		writeLine(out, entity.getName() + "::~" + entity.getName() + "() {};");
 		linebreak(out);
 
-		auto attributes = schema.getAllEntityAttributes(entity);
-
 		// Copy Assignment Operator
 		writeDoxyComment(out, "Assigns the content of \\c other to \\c this.");
 		writeLine(out, entity.getName() + "& " + entity.getName() + "::operator=(const " + entity.getName() + "& other) {");
 		writeLine(out, "this->m_id = other.m_id;");
-		for (auto& attr : attributes) {
+		for (auto& attr : schema.getAllEntityAttributes(entity, true)) {

```

Comment:
Should be again pulled out of the loop because this is probably fairly slow

Reply:
Resolved 3a074828